src/contract.rs

```diff
@@ -746,6 +758,17 @@ fn execute_close_vault(deps: DepsMut, info: MessageInfo) -> Result<Response, Con
     ADDRESSES_WAITING_FOR_EXIT.save(deps.storage, &addresses_waiting_for_exit)?;
     VAULT_TERMINATED.save(deps.storage, &true)?;
 
+    // if no position is open, we can trigger a user update here. otherwise the operator will have to do it when closing the position
+    if JOIN_TIME.load(deps.storage)? == 0 {
```

Comment:
This will always be the case, you are calling this only after forcing exits and you are closing all positions there. You'll always update all users here.